app/products/calls/actions/calls.ts

```diff
@@ -265,6 +266,17 @@ export const joinCall = async (
 
     try {
         await connection.waitForPeerConnection();
+
+        // Follow the thread.
+        const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+        if (!database) {
+            return {data: channelId};
+        }
+
+        const call = getCallsState(serverUrl).calls[channelId];
+        const channel = await getChannelById(database, channelId);
+        updateThreadFollowing(serverUrl, channel?.teamId || '', call.threadId, true, false);
```

Comment:
Do we need some special care if we are the user starting the call? Will the call state, along with the thread id, be available at this point?

Reply:
Yes, even when starting the call, the call state will be available at this point.

Reply:
@streamer45 Hmm... Now you've got me second guessing. We're relying on the await peer connection. Do you think there's a chance there could be a race between the server sending the call started event and the peer connection? Say, a very slow server?

Reply:
@cpoile Right, that's what I was worried about ehe. On the webapp I believe we wait for the start ws event to follow to avoid this exact problem.

Reply:
Ok I'll look into it and make the fix tomorrow. Thanks for double checking that. I'll re-request then.

Reply:
K, good to go now -- added extra race protections, and also checking to make sure we don't follow twice (say on rejoining) -- not strictly necessary, but it reduces redundant API calls.